api/base/urls.py

```diff
@@ -24,7 +24,8 @@
                 url(r'^docs/', include('rest_framework_swagger.urls')),
                 url(r'^institutions/', include('api.institutions.urls', namespace='institutions')),
                 url(r'^collections/', include('api.collections.urls', namespace='collections')),
-                url(r'^guids/', include('api.guids.urls', namespace='guids'))
+                url(r'^guids/', include('api.guids.urls', namespace='guids')),
+                url(r'^identifiers/', include('api.identifiers.urls', namespace='identifiers'))
```

Comment:
Go ahead and add a comma at the end of this item. I know, tiny thing. If this ends up being it for the CR, feel free to ignore. :)